### PR TITLE
feat(mediapro): cloud front integration

### PR DESCRIPTION
**Pull request by ameli90**
https://github.com/JOOlanders/documentation/issues/40

### CODE REVIEW

File: 01.elements/mediapro/02.advanced/04.cloud-front/docs.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    category: docs
+---
+
+## How To Integrate Media Pro with Cloud Front
+
+[CloudFront](https://aws.amazon.com/cloudfront/) Integration is available for JWPlayer and MediaElement players. To enable the integration check **CloudFront Integration** option on the element render settings. 
+ 
+| Field       | Description |
+| :---------- | :---------- |
+| **Distribution** | Set the CloudFront Distribution Domain Name.<br>Ex: `'s1u4jtfxixe2o5'` without `.cloudfront.net.`
```
Comment (miljan-aleksic) [inline 01.elements/mediapro/02.advanced/04.cloud-front/docs.md]: Add some information where this number should be retrieved: "From Amazon CloudFront configuration"

File: 01.elements/mediapro/02.advanced/04.cloud-front/docs.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+title: Integration with Cloud Front
```
Comment (miljan-aleksic) [inline 01.elements/mediapro/02.advanced/04.cloud-front/docs.md]: Fix title: "Amazon CloudFront Integration"

File: 01.elements/mediapro/02.advanced/04.cloud-front/docs.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+title: Integration with Cloud Front 
+taxonomy:
+    category: docs
+---
+
+## How To Integrate Media Pro with Cloud Front
+
+[CloudFront](https://aws.amazon.com/cloudfront/) Integration is available for JWPlayer and MediaElement players. To enable the integration check **CloudFront Integration** option on the element render settings.
```
Comment (miljan-aleksic) [inline 01.elements/mediapro/02.advanced/04.cloud-front/docs.md]: instead of `check` use `toggle`, I think express better the action.

**Comment by ameli90** [comment]
updated

File: 01.elements/mediapro/02.advanced/04.cloud-front/docs.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+## How To Integrate Media Pro with Cloud Front
+
+[CloudFront](https://aws.amazon.com/cloudfront/) Integration is available for JWPlayer and MediaElement players. To enable the integration toggle **CloudFront Integration** option on the element render settings. 
+ 
+| Field       | Description |
+| :---------- | :---------- |
+| **Distribution** | Set the CloudFront Distribution Domain Name from Amazon CloudFront configuration. Ex: `'s1u4jtfxixe2o5'` without `.cloudfront.net.`
+| **Key Pair ID** | Key Pair ID is managed in Amazon Account Manager.
+| **Private Key** | If enabled the track will start playing on load. |
```
Comment (miljan-aleksic) [inline 01.elements/mediapro/02.advanced/04.cloud-front/docs.md]: Fix this description ;)

Comment (ameli90) [inline 01.elements/mediapro/02.advanced/04.cloud-front/docs.md]: You mean Private Key is also from `Amazon CloudFront configuration`? Added

**Comment by miljan-aleksic** [comment]
No, I mean that `If enabled the track will start playing on load` doesn't describe a **private key** at all.

File: 01.elements/mediapro/02.advanced/04.cloud-front/docs.md (hunk 4 above)
Comment (ameli90) [inline 01.elements/mediapro/02.advanced/04.cloud-front/docs.md]: sorry, fixed the Private Key description

**Pull request merged**